test/token-locking.js

```diff
@@ -0,0 +1,179 @@
+      const totalLockCount = await tokenLocking.getTotalTokenLockCount.call(token.address);
+      assert.equal(totalLockCount.toNumber(), 1);
+
+      await colony.waiveRewardPayout(payoutId);
+      const [lockCount] = await tokenLocking.getUserTokenLockInfo.call(token.address, addresses[0]);
+      assert.equal(lockCount.toNumber(), 1);
+    });
+
+    it("should not be able to lock tokens if sender is not colony", async () => {
+      await token.approve(tokenLocking.address, usersTokens);
+      await tokenLocking.deposit(token.address, usersTokens);
+      await checkErrorRevert(tokenLocking.lockToken(token.address), "token-locking-sender-not-colony");
+    });
+
+    it("should not be able to unlock users tokens if sender is not colony", async () => {
+      await token.approve(tokenLocking.address, usersTokens);
+      await tokenLocking.deposit(token.address, usersTokens);
+      await colony.startNextRewardPayout(otherToken.address);
+      await checkErrorRevert(tokenLocking.unlockTokenForUser(token.address, addresses[0]), "token-locking-sender-not-colony");
+    });
+
+    it("should be able to deposit tokens multiple times before if they are unlocked", async () => {
+      await token.approve(tokenLocking.address, usersTokens);
+      await tokenLocking.deposit(token.address, usersTokens / 2);
+      await tokenLocking.deposit(token.address, usersTokens / 2);
+      const userDepositedBalance = await tokenLocking.getUserLockedBalance.call(token.address, addresses[0]);
+      assert.equal(userDepositedBalance.toNumber(), usersTokens);
+    });
+
+    it("should not be able to deposit tokens while they are locked", async () => {
+      await token.approve(tokenLocking.address, usersTokens);
+      await tokenLocking.deposit(token.address, usersTokens);
+      await colony.startNextRewardPayout(otherToken.address);
+      await checkErrorRevert(tokenLocking.deposit(token.address, usersTokens));
+    });
+
+    it("should not be able to withdraw tokens while they are locked", async () => {
+      await token.approve(tokenLocking.address, usersTokens);
+      await tokenLocking.deposit(token.address, usersTokens);
+      await colony.startNextRewardPayout(otherToken.address);
+      await checkErrorRevert(tokenLocking.withdraw(token.address, usersTokens));
+    });
+
+    it("should be able to unlock tokens while they are locked", async () => {
+      await token.approve(tokenLocking.address, usersTokens);
+      await tokenLocking.deposit(token.address, usersTokens);
+      const { logs } = await colony.startNextRewardPayout(otherToken.address);
+      const payoutId = logs[0].args.id;
+      await colony.waiveRewardPayout(payoutId);
+      await tokenLocking.withdraw(token.address, usersTokens);
+      const userDepositedBalance = await tokenLocking.getUserLockedBalance.call(token.address, addresses[0]);
+      assert.equal(userDepositedBalance, 0);
+    });
+
+    it("should not be able to lock tokens twice", async () => {
```

Comment:
You are testing a success here but the title claims you're testing for a failure?

Reply:
Yes, the title is wrong